packages/cozy-sharing/src/components/Recipient/GroupRecipientDetailWithAccess.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+
+import Avatar from 'cozy-ui/transpiled/react/Avatar'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import ToTheCloudIcon from 'cozy-ui/transpiled/react/Icons/ToTheCloud'
+import List from 'cozy-ui/transpiled/react/List'
+import ListItem from 'cozy-ui/transpiled/react/ListItem'
+import ListItemIcon from 'cozy-ui/transpiled/react/ListItemIcon'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
+
+import { getInitials, getDisplayName } from '../../models'
+
+const GroupRecipientDetailWithAccess = ({ withAccess }) => {
+  const { t } = useI18n()
+
+  return (
+    <div>
+      <Typography variant="subtitle2">
+        {t('GroupRecipientDetail.withAccess.title')}
+      </Typography>
+      <List>
+        {withAccess.map((recipient, index) => {
+          const name = getDisplayName(recipient)
+
+          return (
+            <ListItem disableGutters key={index}>
```

Comment:
Same comment as in previous PR, here we should use the recipient's id (the one you created yesterday) instead of the `.map` index.

As a more global rule, we should try to never use `.map` indexes as React keys.

Reply:
Same comment for the `WithoutAccess` component

Reply:
I've corrected the problem, thank you for catching my blunder